tool/system_metrics_monitor.py

```diff
@@ -75,7 +76,11 @@ def publish_metrics_to_firestore(metrics):
 
 
 def run_system_metric_monitor(interval=DEFAULT_INTERVAL):
-    initialize_firebase(sys.argv[1])
+    parser = argparse.ArgumentParser(description='Retrieve system metrics i.e cpu utilization, memory & disk usage')
```

Comment:
Thanks for adding in the description, we don't do this often enough!

Reply:
cool :)